config/locales/en.yml

```diff
@@ -945,14 +946,15 @@ en:
               <p>We calculate the rate based upon the fee scheme, advocate category, offence and case type</p>
         extra_fee_fields:
           net_amount: *net_amount
+          net_amount_html: Net amount <span class="govuk-visually-hidden">in pounds for %{context}</span>
           help_how_we_calculate_rate_title: *how_is_this_calculated_help
           help_how_we_calculate_rate_body: We calculate the amount based upon the fee scheme, case type, offence, advocate category and fee type.
           npw:
             heading_html: Prosecution witnesses
             quantity: Total number of prosecution witnesses
             quantity_hint: *empty_input
           ppe:
-            quantity: Total number of pages
+            quantity: Total number of prosecution evidence pages
```

Comment:
not-a-blocker: "Total number of pages of prosecution evidence" (i.e. PPE) might be a more typical rendering of this term